### PR TITLE
Feature/qt5

### PR DESCRIPTION
**Pull request by timosachsenberg**

### CODE REVIEW

**Comment by timosachsenberg** [comment]
merged develop into qt5 branch.
Had to only resolve a few conflicts:
Conflicts:
	src/openms/source/FORMAT/MascotRemoteQuery.cpp
	src/openms_gui/include/OpenMS/VISUAL/MetaDataBrowser.h
	src/openms_gui/include/OpenMS/VISUAL/ParamEditor.h
	src/openms_gui/include/OpenMS/VISUAL/Spectrum3DOpenGLCanvas.h
	src/openms_gui/include/OpenMS/VISUAL/VISUALIZER/PeptideIdentificationVisualizer.h
	src/openms_gui/include/OpenMS/VISUAL/VISUALIZER/ProteinIdentificationVisualizer.h
	src/openms_gui/source/VISUAL/APPLICATIONS/TOPPASBase.cpp
	src/openms_gui/source/VISUAL/APPLICATIONS/TOPPViewBase.cpp
	src/openms_gui/source/VISUAL/TOPPViewIdentificationViewBehavior.cpp
	src/openms_gui/source/VISUAL/VISUALIZER/ProteinIdentificationVisualizer.cpp
	src/tests/class_tests/openms/source/SVMWrapper_test.cpp
	src/tests/class_tests/openms_gui/source/GUI/TOPPView_test.cpp
	tools/travis/lnx-cibuild.sh
mainly related to nullptr instead of 0.

I overwrote the changes in MascotAdapterRemoteQuery with the ones from develop because I think we need to go over this again with the new test Hannes added.

File: src/openms/include/OpenMS/FORMAT/MascotRemoteQuery.h [hunk 1]
```diff
@@ -156,7 +156,7 @@ private slots:
     String error_message_;
     QTimer timeout_;
     Int to_;
-    Int search_number_;
+    String search_identifier_;
```
Comment (hroest) [inline src/openms/include/OpenMS/FORMAT/MascotRemoteQuery.h]: yes, looks good to me

File: src/openms/source/FORMAT/MascotRemoteQuery.cpp [hunk 2]
```diff
@@ -50,10 +48,10 @@ namespace OpenMS
 
   MascotRemoteQuery::MascotRemoteQuery(QObject* parent) :
     QObject(parent),
-    DefaultParamHandler("MascotRemoteQuery"),
-    manager_(NULL),
-    reply_(NULL)
+    DefaultParamHandler("MascotRemoteQuery")
   {
+    http_ = new QHttp();
```
Comment (hroest) [inline src/openms/source/FORMAT/MascotRemoteQuery.cpp]: I dont think that will compile ... afak there is no more QHttp ... 

I suggest to use the qt5 version with the Mascot 2.6 patch

**Comment by hroest** [comment]
> I overwrote the changes in MascotAdapterRemoteQuery with the ones from develop because I think we need to go over this again with the new test Hannes added.

maybe we should go the other way as now at least it compiles but doesnt do the right thing ... e.g. overwrite with the `feature/qt5` version and then port patches next

**Comment by hroest** [comment]
I tested your version and it works up to a point with some changes. But I can also add these manually after the merge...

**Comment by timosachsenberg** [comment]
ok I updated the contrib and commented out the MAO in a new PR

**Pull request closed without merging**